### PR TITLE
Feature/2599 refactor database

### PR DESCRIPTION
**Pull request by ArronChaveAC**

### CODE REVIEW

File: src/controllers/v2/renewal.js [hunk 1]
```diff
@@ -140,46 +151,41 @@ const RenewalController = {
...
       try {
-        await RegistrationHistory.create(
-          {
-            ...mappedExistingReg,
-            RegistrationId: registrationNumber,
-            createdByLicensingOfficer: existingReg.createdByLicensingOfficer,
-            // Need to confirm expected behaviour here!
-            expiryDate: existingReg.expiryDate
-          },
+        const initialRegistration = await Registration.findByPk(registrationNumber);
```
Comment (RAnderson96) [inline src/controllers/v2/renewal.js:158]: Is this not duplicate of line 104? `const existingReg = findOneRegistrationResponse;`

File: src/controllers/v2/renewal.js [hunk 2]
```diff
@@ -140,46 +151,41 @@ const RenewalController = {
       }
     }
```
Comment (RAnderson96) [inline src/controllers/v2/renewal.js:152]: The portion of the code responsible for detecting changes is now redundant

**Review by GrahamHotchkiss — COMMENTED** [review]
I thought we were going to call the new ID something like `trapHolderId`?

File: src/controllers/v2/registration.js [hunk 3]
```diff
@@ -3,9 +3,6 @@ import {sendSuccessEmail} from '../../notify-emails.js';
 
 const {Registration, Return, NonTargetSpecies, Revocation, Note, RequestUUID} = db;
 
-const REGISTRATION_RENEWAL = 'Renewal';
-const REGISTRATION_INITIAL = 'Initial';
```
Comment (GrahamHotchkiss) [inline src/controllers/v2/registration.js]: @RAnderson96 - Is this now set via the incoming API data? Can only find a get in this code...

Comment (RAnderson96) [reply src/controllers/v2/registration.js]: Yes thats right, its within the payload.

**Pull request merged**